certified-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,965 @@
+    "https"
+  ],
+  "paths": {
+    "/CSSAPI/v2/{entityName}/Add": {
+      "post": {
+        "tags": [
+          "Core Actions"
+        ],
+        "summary": "Create a new a record for a specified entity",
+        "description": "Add a record (limited to one record per call) to specified entity. The response will be the system generated ID.",
+        "operationId": "CSSAPI_AddEntity",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "entityName",
+            "in": "path",
+            "description": "Table name name to add record to.",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Entity Name",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "CSSAPI_EntityListInsert",
+              "value-path": "EntityNameForAPI",
+              "value-title": "EntityNameDisplay"
+            }
+          },
+          {
+            "name": "add",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/APIAddRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        }
+      }
+    },
+    "/CSSAPI/v2/{entityName}/Get": {
+      "post": {
+        "tags": [
+          "Core Actions"
+        ],
+        "summary": "Pull list of record(s) for a given entity",
+        "description": "Pull list of records for a given entity with specific criteria, order by tag, group by tag and number of records. Note: There is no limit on number of records to be pulled but the user has to consider that larger results will be slower to process.",
+        "operationId": "CSSAPI_QueryEntity",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "entityName",
+            "in": "path",
+            "description": "Table name to query data from.",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Entity Name",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "CSSAPI_EntityListSelect",
+              "value-path": "EntityNameForAPI",
+              "value-title": "EntityNameDisplay"
+            }
+          },
+          {
+            "name": "query",
+            "in": "body",
+            "description": "Query Model",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/APIQueryRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Array of Objects",
```

Comment:
@nizarhandal can you please make more precise description for response codes? You can refer [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) for an example.